tests/bluetooth/bsim_bt/edtt_ble_test_app/tests_scripts/ll.set1.test_list

```diff
@@ -35,7 +35,7 @@ LL/CON/CEN/BV-21-C
 LL/CON/CEN/BV-23-C
 LL/CON/CEN/BV-24-C
 LL/CON/CEN/BV-25-C
-LL/CON/CEN/BV-26-C
+#LL/CON/CEN/BV-26-C # This test case is not valid, and will fail with CPR cache
```

Comment:
What is meant by not valid? Is the test case in the test spec broken? If so, is there an erratum for it?

Reply:
The test implementation is incorrect and hence has been disabled for refactored LLCP testing too, here: https://github.com/zephyrproject-rtos/zephyr/blob/617dacfdccd007e611895c007ef345e8cf34889b/tests/bluetooth/bsim_bt/edtt_ble_test_app/tests_scripts/ll.set1.llcp.test_list#L38